insecure/corruptlibp2p/gossipsub_spammer.go

```diff
@@ -0,0 +1,50 @@
+package corruptlibp2p
```

Comment:
Spammer that will be used for future spam tests of gossipsub - for now, it just implements IHAVE control message spamming